doc/build.md

```diff
@@ -15,6 +15,22 @@ Please note that all mining functions for Grin have moved into a separate, stand
 [grin_miner](https://github.com/mimblewimble/grin-miner). Once your Grin code node is up and running,
 you can start mining by building and runing grin-miner against your running Grin node.
 
+## Docker
+
+        # Build using all available cores
+        docker build -t grin .
+
+        # or build using a specific number of cores (reduce RAM requirement)
+        docker build --build-arg NPROC=1 -t grin .
```

Comment:
Do you think this is going to be used frequently? I would boil these instructions down to a single docker build & run with -d.

Reply:
Yeah I can probably remove the ```NPROC``` part. However, I think running in foreground and background is nice.

Reply:
Okay I simplified a bit this part and added a bit of explanation.